### PR TITLE
Allow wc-admin plugin to be installed for new OBW.

### PR DESCRIPTION
**Pull request by timmyc**
Just a try branch to see what is needed to allow `wc-admin` plugin to still be installed during the OBW.

### CODE REVIEW

File: includes/admin/class-wc-admin-setup-wizard.php [hunk 1]
```diff
@@ -133,13 +133,16 @@ protected function should_show_facebook() {
...
+	 * Is the WooCommerce Admin Plugin installed already?
 	 *
 	 * @return boolean
 	 */
-	protected function is_wc_admin_active() {
-		return function_exists( 'wc_admin_url' );
+	protected function is_wc_admin_plugin_active() {
+		$is_plugin_active = false;
+		if ( class_exists( 'Automattic\WooCommerce\Admin\Compose\Package' ) ) {
+			$is_plugin_active = Automattic\WooCommerce\Admin\Compose\Package::is_package_active();
```
Comment (psealock) [inline includes/admin/class-wc-admin-setup-wizard.php]: This seems off. If `is_package_active()` returns true, that would mean the plugin is **in**active.

Comment (jeffstieler) [reply includes/admin/class-wc-admin-setup-wizard.php]: Why not use [`is_plugin_active()`](https://developer.wordpress.org/reference/functions/is_plugin_active/) here?

**Comment by jeffstieler** [comment]
In testing this, I got the prompt for the new OBW, but it kept me in the old wizard despite installing the WC-Admin plugin:



However, I've supplanted this PR with: https://github.com/woocommerce/woocommerce/pull/25842

**Comment by timmyc** [comment]
closing in favor of #25842

**Pull request closed without merging**